.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Great work adding the `.gitignore` file to your project. However, following the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) it seems that you created and pushed the `test.md` file to your repository.
To resolve this issue, please delete the `test.md` file add it to your `.gitignore` file, and push your changes again. This should ensure that the file is properly ignored and not pushed to the repository. 💪

  ![image](https://github.com/binyamolango/Hello-Microverse/assets/107952724/f77b7c37-183c-43bb-82be-35f5488af408)

Reply:
@taldr27 I addressed this issue too, I am about to pull another request. Thank you so much for the feedback.

Reply:
@binyamolango you're welcome, you can push your changes and ask for a new review using this same Pull Request. 👌

Reply:
@taldr27 I just did. Thanks 😉